core/app/c/[communitySlug]/pubs/[pubId]/page.tsx

```diff
@@ -132,35 +125,29 @@ export default async function Page({
 					<div>
 						<div className="mb-1 text-lg font-bold">Current Stage</div>
 						<div className="ml-4 flex items-center gap-2 font-medium">
-							<div>
-								{pub.stages.map(({ stage }) => {
-									return (
-										<div key={stage.id} data-testid="current-stage">
-											{stage.name}
-										</div>
-									);
-								})}
-							</div>
-							{pub.stages[0] ? (
```

Comment:
the new getPub query limits stage selection to 1, so we no longer have to work with indexing the first item in an array